superset/migrations/versions/de021a1ca60d_.py

```diff
@@ -0,0 +1,22 @@
+"""empty message
+
+Revision ID: de021a1ca60d
+Revises: ('0b1f1ab473c0', 'cefabc8f7d38')
+Create Date: 2018-12-18 22:45:55.783083
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = 'de021a1ca60d'
+down_revision = ('0b1f1ab473c0', 'cefabc8f7d38')
```

Comment:
Alembic can't merge 3 heads into 1?

Reply:
Oh I'm sure it does, I just rebased 2 times, let me collapse these